zuul.d/pipelines.yaml

```diff
@@ -50,6 +50,7 @@
       github:
         status: pending
         comment: false
+        status-url: https://status.openlabtesting.org/status
```

Comment:
all pipelines should add this as well IMO.

Reply:
done